Exercise_2.java

```diff
@@ -8,11 +8,24 @@
        of pivot */
     void swap(int arr[],int i,int j){
         //Your code here   
+        int temp = arr[i];

```

Comment:
Good Job! Please add time and space complexity and comments explaining your approach.